packages/web/src/pages/upload-page/forms/AttributionModalForm.tsx

```diff
@@ -262,6 +276,8 @@ const AttributionModalFields = () => {
             <SegmentedControl
               defaultSelected={allowAttribution}
```

Comment:
so typically should remove "defaultValue" if using a controlled value. not sure if aware but `defaultValue` is a way to set a default for an uncontrolled component.

Reply:
yeah makes sense. A bunch of these were "required" but I bet they function fine when passing a value